Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
+    private StringBuilder logString;
+    public string Label { get; set; }
+
+    private LogType logType;
+    private bool logCommonColumns;
+
+    private int nbLines;
+    private bool createStringOverTime;
+    private StringBuilder currentLineLogged;
+    private SortedDictionary<string, string> currentLogs;
+    private const string fieldSeparator = ";";
+    private const string lineSeperator = "\n";
+    private string email;
+    public string SessionId { get; set; }
+
+    public LogStore(string label, string email, bool createStringOverTime,
+        LogType logType = LogType.Normal, bool logCommonColumns = true)
+    {
+        Init(label, email, Guid.NewGuid().ToString(), createStringOverTime, logType, logCommonColumns);
+    }
+
+    public LogStore(string label, string email, string sessionID, bool createStringOverTime,
+        LogType logType = LogType.Normal, bool logCommonColumns = true)
+    {
+        Init(label, email, sessionID, createStringOverTime, logType, logCommonColumns);
+    }
+
+    private void Init(string label, string email, string sessionID, bool createStringOverTime,
+        LogType logType, bool logCommonColumns)
+    {
+        this.Label = label;
+        logs = new SortedDictionary<string, List<string>>();
+        logString = new StringBuilder();
+        currentLineLogged = new StringBuilder();
+        currentLogs = new SortedDictionary<string, string>();
+        this.createStringOverTime = createStringOverTime;
+        this.email = email;
+        SessionId = sessionID;
+        this.logType = logType;
+        this.logCommonColumns = logCommonColumns;
+        if (logType == LogType.Normal && logCommonColumns)
+        {
+            logs.Add("Timestamp", new List<string>());
+            logs.Add("Framecount", new List<string>());
+            logs.Add("SessionID", new List<string>());
+            logs.Add("Email", new List<string>());
+        }
+        else if (logType == LogType.Meta)
+        {
+            logs.Add("SessionID", new List<string>());
+            logs.Add("Email", new List<string>());
+        }
```

Comment:
FYI, I guess right now it is possible to disable `logCommonColumns` but still ask for a `Meta` logType. But I suggest we just remove the `logCommonColumns` altogether. (see my other comment)